TOOLS/osxbundle.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-
+from __future__ import print_function
```

Comment:
python 2 is dead anyway though, are you sure this is needed and the additional space between `print` and `(` wasn't just a typo?

Reply:
this is not necessary for the change on line 40, but I saw this  `from __future__ import ` from other files, so I assume a py2 compatibility statement is required for this repo.   I can remove this line.

Reply:
removed in https://github.com/mpv-player/mpv/pull/7354/commits/9ad9f1d6d3365f6d5f76d4da9ed841075a13ecfd